src/threeIdConnect.ts

```diff
@@ -66,13 +65,13 @@ class ThreeIdConnect {
    *
    * @param     {String}    iframeUrl   iframe url, defaults to 3id-connect iframe service
    */
-  constructor(iframeUrl: string) {
+  constructor(iframeUrl?: string) {
```

Comment:
Would be nice if we could tell this constructor which ceramic network we are using, like `testnet-clay`, or `mainnet`. That would make the developer experience much better.

Reply:
yeah good call, that means we will have one deployed on each network, app-dev.3idconnect.org can be on testnet clay after, not changing here, but will add issue